### PR TITLE
Add some more linting rules

### PR DESCRIPTION
**Pull request by markgaze**
This is a start, I've moved over some rules we usually use and ones that I've used on other Next projects.

We can keep building them up and there's a few areas of the code that have warnings left but we can get through them eventually!

### CODE REVIEW

**Comment by markgaze** [comment]
#### 📝Changed routes:


| [**/author/\[id\]**](https://sealwadding-blog-6wjbl2wsp-markgaze.vercel.app/author/[id]) | [**/authors**](https://sealwadding-blog-6wjbl2wsp-markgaze.vercel.app/authors) |
|:-:|:-:|
|<a href="https://sealwadding-blog-6wjbl2wsp-markgaze.vercel.app/author/[id]">Screenshot of /author/[id]</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/D13_DxIIpAP0-5FMDgYnU8NbCy0TKtyWly5L6R5mJ2_kJt3l3lZLiCvO1SFysalT7Ba7IeXwoaScJlyc7ouDRQ.png">(view full size)</a> |<a href="https://sealwadding-blog-6wjbl2wsp-markgaze.vercel.app/authors">Screenshot of /authors</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/fonzDY-d0JiMkD3f--QLc0s31SBGfgqJLPrFQxlYbvuESRPcjTmk6t15mLHjpMEdtBvQCKrufzQ5cGeaWD1tOw.png">(view full size)</a> |



| [**/**](https://sealwadding-blog-6wjbl2wsp-markgaze.vercel.app/) | [**/post/\[id\]**](https://sealwadding-blog-6wjbl2wsp-markgaze.vercel.app/post/[id]) |
|:-:|:-:|
|<a href="https://sealwadding-blog-6wjbl2wsp-markgaze.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/yuKCKBjjpn2EThtd6bK0Q9L3GLUb73q19nU_SmBalQDC3eSW-_2yUNCGrCq6ySUpC-msUZAzioko0wU9s978oA.png">(view full size)</a> |<a href="https://sealwadding-blog-6wjbl2wsp-markgaze.vercel.app/post/[id]">Screenshot of /post/[id]</a><br /><sup><a href="https://images.weserv.nl?url=https://iad.microlink.io/gZ6-QzkBhjEiJtxv20xGpCP2z3DeVV7oYhfiUa5g0HHxh3ZpuLohOZwYssLBxJ1VBHoXdZ5o2cQfZb3FOuZ5wA.png">(view full size)</a> |



And 1 other route:
- [**/api/hello**](https://sealwadding-blog-6wjbl2wsp-markgaze.vercel.app/api/hello)



Commit 3670ce49ec782e5fda33946e7deffaeb3df315cb (https://sealwadding-blog-6wjbl2wsp-markgaze.vercel.app).

File: .stylelintrc [hunk 1]
```diff
@@ -1,3 +1,14 @@
 {
-  "extends": ["stylelint-config-standard", "stylelint-config-prettier"]
+  extends: ['stylelint-config-sass-guidelines', 'stylelint-config-prettier'],
+  plugins: ['stylelint-order'],
+  rules: {
+    'selector-class-pattern': [
```
Comment (andrewbarnesweb) [inline .stylelintrc:5]: Reckon we can ditch this, as we're using CSS Modules 🙂

**Review by andrewbarnesweb — CHANGES_REQUESTED** [review]
Holy smokes! You have been busy!

**Comment by markgaze** [comment]
I accidentally included the `.vscode` settings folder so if you want me to take it out, I can but I know we've committed it to some repos

**Comment by andrewbarnesweb** [comment]
Nah looks good. Not looking forward to rebasing onto this though 😂

**Review by andrewbarnesweb — APPROVED** [review]

**Pull request merged**